### DISCUSSION TITLE
how do you upload files to GitHub?

### DISCUSSION THREAD

**Discussion by NotNORMAN5** [community/community#40143] [New to GitHub]
I need help uploading a file to GitHub

**Comment by ossanna16** [comment]
Hi there @NotNORMAN5 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by AlefMemTav** [comment]
See it: https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository

**Comment by jfullstackdev** [comment]
GitHub Web also has the drag and drop, like any other drag and drop feature
